tests/sentry/integrations/source_code_management/test_commit_context_slo.py

```diff
@@ -20,6 +21,7 @@ class MockCommitContextIntegration(CommitContextIntegration):
 
     def __init__(self):
         self.client = Mock()
+        self.client.base_url = "https://example.com"
```

Comment:
not to make more work but would be cool to have this be the appropriate type (`CommitContextClient`)